test/subset/data/tests/sbix.tests

```diff
@@ -0,0 +1,13 @@
+FONTS:
+sbix.ttf
+
+PROFILES:
+default.txt
+drop-hints.txt
+drop-hints-retain-gids.txt
+retain-gids.txt
+name-ids.txt
```

Comment:
You can leave out name-ids.txt it should have no effect on sbix subsetting.

Reply:
Done.